apps/dev/src/commands/feeds/read/adfs.ts

```diff
@@ -91,10 +139,23 @@ export const adfs = Command.make(
           );
           data.forEach((word, i) => rows.push([`Data[${i}]`, word]));
         } else {
-          const data = yield* Effect.tryPromise(() =>
+          const single = yield* Effect.tryPromise(() =>
             consumer.getSingleDataAtIndex(feed, index.value),
           );
-          rows.push(['Data', data]);
+          if (isHumanReadable && !hasSlice && !multi) {
+            const parsed = parseADFSData(single);
+            rows.push(['Data (Raw)', parsed.rawHex]);
+            rows.push(['Value', parsed.value]);
+            rows.push(['Value (Hex)', parsed.valueHex]);
+            if (parsed.timestamp) {
+              rows.push(['Timestamp', parsed.timestamp]);
+              rows.push(['Timestamp (Hex)', parsed.timestampHex ?? '']);
+              if (parsed.formattedTimestamp)
+                rows.push(['Timestamp (Formatted)', parsed.formattedTimestamp]);
```

Comment:
With all the changes above the logic of the table rows is that simple:
```suggestion
            rows.push(['Data (Raw)', parsed.rawHex]);
            rows.push(['Value', parsed.value]);
            rows.push(['Timestamp', parsed.timestamp]);
```